Android/app/src/main/java/app/intra/net/doh/ServerConnectionFactory.java

```diff
@@ -43,14 +44,10 @@ public ServerConnectionFactory(Context context) {
   /**
    * @return True if these URLs represents the same server.
    */
-  public static boolean equalUrls(String url1, String url2) {
-    // null and "" are equivalent, representing the default server.
-    if (url1 == null) {
-      url1 = "";
-    }
-    if (url2 == null) {
-      url2 = "";
-    }
+  public boolean equalUrls(String url1, String url2) {
```

Comment:
I wonder if this method better remain static.

Reply:
OK, done.